drivers/input/i8042prt/hwhacks.c

```diff
@@ -271,6 +335,10 @@ i8042InitializeHwHacks(
         return;
     }
 
+    /* Store SMBios data in registry */
```

Comment:
Add comment: this function should be removed once the `mssmbios` driver/service is implemented.